src/elements/modifiers/Accessibility/Accessibility.container.js

```diff
@@ -45,6 +46,63 @@ export const Accessibility = (el) => {
       test: '[role="link"]',
       rule: (element) => (element.ariaLabel),
       error: (element) => `Missing "aria-label" prop (${element.parentNode.innerHTML}):\r\nWhen transforming a Link element's tagName to anything other than <a>, you need to supply an "arial-label" attribute and describe the elements intent to screen readers.`
+    },
+    // Test for buttons without text whose only child is an icon who have missing aria-hidden
+    {
+      test: ['.button--icon'],
```

Comment:
I think this needs to be passed as a string, not array.
Lets update this to just be '.button--icon' without the hard brackets.

Nice work btw!